packages/plasma-new-hope/src/examples/plasma_b2c/components/Tree/Tree.config.ts

```diff
@@ -21,6 +22,10 @@ export const config = {
                 ${tokens.checkboxBorderColor}: var(--text-secondary);
                 ${tokens.checkboxBackgroundColor}: var(--text-accent);
                 ${tokens.itemDisabledOpacity}: 0.4;
+                ${tokens.itemDisabledBorderColor}: var(--text-secondary);
+                ${tokens.itemDisabledColor}: var(--text-secondary);
+                ${tokens.itemDisabledBorderColor}: var(--text-secondary);
+                ${tokens.itemDisabledColor}: var(--text-secondary);
```

Comment:
задублировались токены

```suggestion
                ${tokens.itemDisabledBorderColor}: var(--text-secondary);
                ${tokens.itemDisabledColor}: var(--text-secondary);
```